src/Data/Units/Speed.purs

```diff
@@ -0,0 +1,11 @@
+module Data.Units.Speed where
```

Comment:
Can we please rename this module to `Nautical`? It contains nautical miles as well, not just units of speed.